cmd/osv-scanner/internal/helper/helper.go

```diff
@@ -22,10 +22,10 @@ import (
 // OfflineFlags is a map of flags which require network access to operate,
 // with the values to set them to in order to disable them
 var OfflineFlags = map[string]string{
-	"include-git-root":                     "true",
-	"experimental-offline-vulnerabilities": "true",
-	"experimental-no-resolve":              "true",
-	"experimental-licenses-summary":        "false",
+	"include-git-root":              "true",
```

Comment:
I'm not sure when this got added, but let's remove include-git-root as getting git root doesn't do anything when offline.

Reply:
removed